WooCommerce/src/main/kotlin/com/woocommerce/android/ui/orders/details/OrderDetailViewModel.kt

```diff
@@ -623,12 +630,20 @@ class OrderDetailViewModel @Inject constructor(
     }
 
     private fun fetchShipmentTrackingAsync() = async {
+        pluginsInformation[WooCommerceStore.WooPlugin.WOO_SHIPMENT_TRACKING.pluginName]?.let { plugin ->
+            // Fetch data only when the plugin is installed and active
+            if (!plugin.isInstalled || !plugin.isActive) return@async
+        }
         val result = orderDetailRepository.fetchOrderShipmentTrackingList(navArgs.orderId)
         appPrefs.setTrackingExtensionAvailable(result == SUCCESS)
         orderDetailsTransactionLauncher.onShipmentTrackingFetched()
     }
```

Comment:
Very nitpicky on my part. Feel free to ignore it if you want 😬, but maybe we could improve a little bit this semantic by placing the constraint rule in a `takeIf` call since we're already using a `let` chaining to acquire the plugin:

```suggestion
               pluginsInformation[WooCommerceStore.WooPlugin.WOO_SERVICES.pluginName]
            ?.takeIf { it.isInstalled && it.isActive }
            ?.let {
                orderDetailRepository.fetchOrderShippingLabels(navArgs.orderId)
                orderDetailsTransactionLauncher.onShippingLabelFetched()
        }
```

WDYT?